lib/core/utils/collect-results-from-frames.js

```diff
@@ -25,12 +25,19 @@ function collectResultsFromFrames(
 
   // Tell each axe running in each frame to collect results
   frames.forEach(frame => {
+    const tabindex = frame.node.getAttribute('tabindex');
+    const focusable = tabindex ? parseInt(tabindex) >= 0 : true;
+
     var params = {
       options: options,
       command: command,
       parameter: parameter,
       context: {
         initiator: false,
+
+        // if any parent iframe is not focusable then the entire
+        // iframe stack is not focusable
+        focusable: context.focusable === false ? false : focusable,
```

Comment:
Better to have the code explain itself then to add comments. We can solve this by changing the variable name.

```suggestion
        focusable: parentFrameContext.focusable === false ? false : focusable,
```